spec.html

```diff
@@ -26924,31 +26924,37 @@ <h1>Native Error Types Used in This Standard</h1>
 
       <emu-clause id="sec-native-error-types-used-in-this-standard-evalerror">
         <h1>EvalError</h1>
+        <p>Is the intrinsic object <dfn>%EvalError%</dfn>.</p>
```

Comment:
We don't typically do this as a sentence/paragraph on its own. We do it in an unordered list. We could either switch to an unordered list here or (my preference) make this a complete sentence and tack it on to the following paragraph.